examples/react/messenger/index.js

```diff
@@ -893,4 +904,90 @@ function listMembers(members){
 			list.push(member.name);
         })
 	return list.join(', ');
+}
+
+// MonkeyChat: Push
+
+function createPush(conversationId, pushKey) {
+
+	const username = store.getState().users.userSession.name;
+    let pushLocalization;
+    let text;
+	let locArgs;
+
+    if (!isConversationGroup(conversationId)) {
+	    locArgs = [username];
+        switch(pushKey) {
+            case 1: // text message
```

Comment:
is there no clean way to have an enum or something for these magic numbers? @erikaperugachi